src/hooks/useCourtContracts.js

```diff
@@ -206,6 +210,25 @@ export function useDisputeActions() {
     [disputeManagerContract, processRequests]
   )
 
+  const autoReveal = useCallback(
```

Comment:
Should we add a comment header here, since it doesn't really belong under "draft jurors"? Maybe "request auto reveal"?

Reply:
Yeah definitely